setup_wizard/character_rig_setup/rig_script.py

```diff
@@ -939,17 +953,8 @@ def move_into_collection(object,collection):
     if camera_coll:
         bpy.data.collections.remove(camera_coll,do_unlink=True)
 
-    # Let's 'exclude' that wgt collection: https://blenderartists.org/t/disable-exlude-from-view-layer-in-collection/1324744
-    def recurLayerCollection(layerColl):
-        found = None
-        if (layerColl.name == "wgt"):
-            return layerColl
-        for layer in layerColl.children:
-            found = recurLayerCollection(layer)
-            if found:
-                return found
     layer_collection = bpy.context.view_layer.layer_collection
-    layerColl = recurLayerCollection(layer_collection)
+    layerColl = searchForLayerCollection(layer_collection, "wgt")
     bpy.context.view_layer.active_layer_collection = layerColl
 
     layerColl.exclude = True

```

Comment:
Not sure if we can leverage the new `disable_collection()` method or if the `active_layer_collection` is important for the rest of the script.

But I'm fine if we leave it be and don't touch it